cmd/prometheus/main.go

```diff
@@ -85,12 +85,15 @@ func Main() int {
 		log.Errorf("Error initializing remote storage: %s", err)
 		return 1
 	}
-
 	if remoteStorage != nil {
 		sampleAppender = append(sampleAppender, remoteStorage)
 		reloadables = append(reloadables, remoteStorage)
 	}
 
+	configurableRemoteStorage := remote.NewConfigurable()
```

Comment:
Maybe call it "reloadable" instead of "configurable", as that's been a term we've used at least in main to describe reloadables.

When we remove the other write paths, I imagine we will drop the reloadable/configurable part of the name again.